intercom/db/spatch.py

```diff
@@ -0,0 +1,50 @@
+from intercom.db.database import DatabaseInterface
+
+import requests
+
+from intercom.db.errors import ServerUnreachableError, ServerVersionUnrecognisedError, DatabaseError
+
+
+class SpaTchDatabase(DatabaseInterface):
```

Comment:
Could add an intermediate class 
```suggestion
class SpaTchDatabase(WebInterface):

class WebInterface(DatabaseInterface):
```

With `WebInterface` implementing base method to fetch on defined endpoints and `SpaTchDatabase` setting those endpoints and a custom parser function. 

This architecture is kinda heavy tho and since we probably won't have any other `DatabaseInterface` and even less other `WebInterface`.

Reply:
Agree on the fact it may not be usefull now, and we are good to go with only `class SpaTchDatabase(DatabaseInterface):`